controller/salesemployeeController.ts

```diff
@@ -0,0 +1,20 @@
+import { Application, Request, Response } from "express";
+import { SalesEmployee } from "../model/salesemployee"; 
+
+const salesemployeeService = require('../service/salesemployeeService')
+
+
+module.exports = function(app: Application){
```

Comment:
There is a more Typescript/user friendly way to handle this export

Instead of module.export

```
export const salesEmployeeController = (app: Application) => {
     ...
}

```

The app.ts file could then import the controller with the following syntax:
`import { salesEmployeeController } from 'path`

and then you can call the function by doing
`salesEmployeeController(app)`